packages/bierzo-wallet/src/routes/transactions/components/TxTable/TxTableHeader/utils/TxColumn.tsx

```diff
@@ -0,0 +1,36 @@
+import { makeStyles } from '@material-ui/core';
+import classNames from 'classnames';
+import Block from 'medulas-react-components/lib/components/Block';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import * as React from 'react';
+
+const useStyles = makeStyles({
+  header: {
```

Comment:
Can be moved to `Block` but we need to figure it out what to do with `flexBasis`